pom.xml

```diff
@@ -48,6 +48,10 @@
     </issueManagement>
 
     <properties>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.release>11</maven.compiler.release>
```

Comment:
Are those properties really needed here? Or ist this more a "just in case" thing? I'm surely missing something here...

Reply:
Yes, because jboss-parent sets JDK 8 as the source/target version

Reply:
Ok, but which module that is actually compiling something is _not_ an independent-project and does not inherit from `build-parent`? Are there any?

Reply:
`build-parent` and `bom/application` and `bom/test` inherit from `quarkus-parent`. Looks like I should remove from build-parent